linalg/strumpack.hpp

```diff
@@ -41,20 +43,26 @@ class STRUMPACKRowLocMatrix : public Operator
                          int *I, HYPRE_BigInt *J, double *data,
                          bool sym_sparse = false);
 
-   /** Creates a copy of the parallel matrix hypParMat in STRUMPACK's RowLoc
-       format. All data is copied so the original matrix may be deleted. */
+   /** @brief Creates a copy of the parallel matrix hypParMat in STRUMPACK's RowLoc
+       format.
+
+       All data is copied so the original matrix may be deleted.
+   */
    STRUMPACKRowLocMatrix(const Operator &op, bool sym_sparse = false);
 
    ~STRUMPACKRowLocMatrix();
 
+   /// Matrix vector products are not supported on for this type of matrix.
    void Mult(const Vector &x, Vector &y) const
    {
       MFEM_ABORT("STRUMPACKRowLocMatrix::Mult: Matrix vector products are not "
                  "supported!");
    }
 
+   /// Get the MPI Comm being used by the parallel matrix
    MPI_Comm GetComm() const { return A_->comm(); }
 
+   /// Gain access to the internal CSR matrix
```

Comment:
pedantic: Maybe "Gain" -> "Get" access to be consistent with the function name "GetA"